src/components/Gnb.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import './Gnb.css';
+import logoImg from '../assets/logo.svg';
+
+const Gnb = ({className, items}) => {
+  const isProfileAvailable = items && items.profileImageSource && items.email;
```

Comment:
사소하지만 화면에 보여줄지, 안보여줄지를 결정하는 값이라 show같은 이름은 어떨까 싶네요~
```suggestion
  const isShowProfile = items && items.profileImageSource && items.email;
```